RecoTracker/SpecialSeedGenerators/python/cosmicDC_cff.py

```diff
@@ -0,0 +1,36 @@
+import FWCore.ParameterSet.Config as cms
+
+# seeding
+import RecoTracker.SpecialSeedGenerators.outInSeedsFromStandaloneMuons_cfi
+import TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi 
+hitCollectorForOutInMuonSeeds = TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi.Chi2MeasurementEstimator.clone(
```

Comment:
this has to be renamed to something else, e.g. hitCollectorForCosmicDCMuonSeeds
(the  ComponentName will need to be updated as well as hitCollector inside of the cosmicDCSeeds below.
Current version overrides the definition from RecoTracker/SpecialSeedGenerators/python/outInSeedsFromStandaloneMuons_cfi.py